_posts/2020-06-17-ombulabs-open-source-guidelines.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "OmbuLabs Open Source Guidelines"
+date: 2020-06-17 11:16:00
+categories: ["open-source", "guideline"]
+author: bronzdoc
+---
+
+![goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg](https://tettra-production.s3.us-west-2.amazonaws.com/0d6efb4f154041e899af17bdcd19c1b5/ce8d5f452da02a88554d80c256162a35/d822b155a4112474fdb7aea5ee22465e/d28647d6b4d6e5842508679707ee21a7/goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg)
+
+At OmbuLabs we dedicate some of our work hours to open source every week. Also some of us contribute to open source during our spare time.
```

Comment:
@FionaDL  what you think?  "we dedicate some hours to open source. " ?

Reply:
@bronzdoc it would have to be "we dedicate some of our hours to open source" Maybe:
"At OmbuLabs some of our time is spent working on open source projects."